src/components/graphic.js

```diff
@@ -1,35 +1,41 @@
 import React from "react";
-import { Grid } from "@trussworks/react-uswds";
+import { Grid, GridContainer } from "@trussworks/react-uswds";
```

Comment:
I added GridContainer back in because it creates spacing/formatting that aligns with the USWDS template. The code example in the trustworks template also uses multiple GridContainer components.